formfactory/forms.py

```diff
@@ -59,3 +59,25 @@ class FormActionParamAdminForm(forms.ModelForm):
     class Meta:
         model = models.ActionParam
         fields = ["key", "value"]
+
+
+class WizardAdminForm(forms.ModelForm):
+    class Meta:
+        model = models.Wizard
+        fields = ["title", "slug", ]
```

Comment:
Redundant trailing comma in list, this is only typical of a one item tuple, i.e `("item 1", )`